tariff-management-service/api/src/main/java/com/microservices/tariffmanagement/api/service/TariffService.java

```diff
@@ -0,0 +1,11 @@
+package com.microservices.tariffmanagement.api.service;
+
+import com.microservices.tariffmanagement.api.dto.TariffDto;
+
+import java.util.List;
+
+public interface TariffService {
```

Comment:
1. Make sure this TariffService is not that you're implementing in app package 
2. DOCS

Reply:
fixed